src/java.desktop/macosx/native/libawt_lwawt/awt/OSVersion.m

```diff
@@ -33,6 +33,23 @@
 
 // returns 107 for Lion, 106 for SnowLeopard etc.
 int getOSXMajorVersion() {
+
```

Comment:
this is dead code, it used to support some workarounds for snow leopard. As snowleopard is completely unsupported in openjdk16, it's much better to completely remove this dead code (whole OSVersion.*)
isSnowLeopardOrLower can just always return true, or removed and it's use in code can be simplified